### ISSUE TITLE
Explain to tensor

### ISSUE THREAD

**Issue by Optimox** [dreamquark-ai/tabnet#45]
## Feature request
Currently output of explain is of tensor format, should be of numpy.

**What is the expected behavior?**
Should be numpy array

**What is motivation or use case for adding/changing the behavior?**

Everyone expects numpy arrays
**How should this be implemented in your opinion?**
.detach().numpy()

**Are you willing to work on this yourself?**
yes
